### PR TITLE
Lance/boo joint v2

### PR DESCRIPTION
**Pull request by LanceUp**
changes in naming

### CODE REVIEW

File: contracts/BooJoint.sol [hunk 1]
```diff
@@ -22,7 +22,7 @@ interface IERC20Extended {
     function symbol() external view returns (string memory);
 }
 
-contract BooJoint {
+contract BooJointWftmBoo {
```
Comment (poolpitako) [inline contracts/BooJoint.sol:25]: Plz keep the `BooJoint` name.

File: contracts/BooJoint.sol [hunk 2]
```diff
@@ -226,7 +229,7 @@ contract BooJoint {
     }
 
     function pendingReward() public view returns (uint256) {
-        return masterchef.pendingBOO(pid, address(this));
+        return masterchef.pendingCake(pid, address(this));
```
Comment (poolpitako) [inline contracts/BooJoint.sol:232]: it needs to be `.pendingBOO`

**Review by poolpitako — CHANGES_REQUESTED** [review]

File: contracts/BooJoint.sol (hunk 2 above)
Comment (LanceUp) [reply contracts/BooJoint.sol:232]: done in V3, thanks!

File: contracts/BooJoint.sol (hunk 1 above)
Comment (LanceUp) [reply contracts/BooJoint.sol:25]: done in V3, thanks!

File: contracts/BooJoint.sol [hunk 3]
```diff
@@ -226,7 +229,7 @@ contract BooJoint {
     }
 
     function pendingReward() public view returns (uint256) {
-        return masterchef.pendingBOO(pid, address(this));
+        return masterchef.pendingBoo(pid, address(this));
```
Comment (poolpitako) [inline contracts/BooJoint.sol:232]: pretty sure this is case sensitive. use `.pendingBOO`

**Review by poolpitako — APPROVED** [review]

**Pull request merged**